regolith/helper.py

```diff
@@ -21,8 +21,7 @@
     "l_grants": (l_grants.GrantsListerHelper, l_grants.subparser),
     "l_members": (l_members.MembersListerHelper, l_members.subparser),
     "l_contacts": (l_contacts.ContactsListerHelper, l_contacts.subparser),
-    "u_logurl": (u_logurl.LogUrlUpdaterHelper, u_logurl.subparser)
-    "u_contact": (u_contact.ContactUpdaterHelper, u_contact.subparser),
```

Comment:
did you just delete someone's contact updater helpers?

Reply:
It wasn't originally in my code even after I git pulled upstream master but github was showing it as a conflict that needed to be resolved so I added it in. But then I wasn't sure if that was the right way to go about it so I removed it again...

Reply:
Nevermind! I fixed the conflicts and merged master into my branch so that it is all up to date!